consensus/metrics.go

```diff
@@ -81,12 +86,24 @@ func PrometheusMetrics(namespace string, labelsAndValues ...string) *Metrics {
 			Name:      "validators",
 			Help:      "Number of validators.",
 		}, labels).With(labelsAndValues...),
+		LastSignedHeight: prometheus.NewGaugeFrom(stdprometheus.GaugeOpts{
+			Namespace: namespace,
+			Subsystem: MetricsSubsystem,
+			Name:      "last_signed_height",
+			Help:      "Last signed height for a validator",
+		}, append(labels, "address")).With(labelsAndValues...),
 		ValidatorsPower: prometheus.NewGaugeFrom(stdprometheus.GaugeOpts{
 			Namespace: namespace,
 			Subsystem: MetricsSubsystem,
 			Name:      "validators_power",
 			Help:      "Total power of all validators.",
 		}, labels).With(labelsAndValues...),
+		ValidatorPower: prometheus.NewGaugeFrom(stdprometheus.GaugeOpts{
+			Namespace: namespace,
+			Subsystem: MetricsSubsystem,
+			Name:      "validator_power",
+			Help:      "Power of a validator.",
+		}, append(labels, "address")).With(labelsAndValues...),
```

Comment:
don't you think address is too abstract? maybe

```suggestion
		}, append(labels, "validator_address")).With(labelsAndValues...),
```

Reply:
thought of this right when I pushed and made the change locally then switched branches and deleted the change